apps/svelte.dev/src/routes/packages/packages-meta.ts

```diff
@@ -0,0 +1,312 @@
+import type { Package } from '$lib/server/content';
+
+const TAGS = {
+	'design-system': {
+		prompt:
+			'MUST contain 2+ reusable Svelte components designed as a cohesive set with shared design principles. Examples: complete UI kits, design systems, form libraries, widget collections, component libraries. Key indicator: components are intentionally built to work together, share styling patterns, and serve a unified purpose. NOT a single component or unrelated components bundled together.'
+	},
```

Comment:
> The scripts for updating the registry have also not been copied over

Maybe `TAGS` is not needed?

Reply:
thanks. removed for now. we can add it back if needed, but there was a lot of hesitancy around the AI portion, so I'm just leaving it out for now until/unless it's something agreed upon